src/util/submit-addon.js

```diff
@@ -66,6 +66,7 @@ type ClientConstructorParams = {|
   approvalCheckInterval?: number,
   approvalCheckTimeout?: number,
   downloadDir?: string,
+  userAgentString?: string,
```

Comment:
I think we should make this argument mandatory, not optional and without a default value.

Reply:
it has a default value (`'webext'`) defined in the constructor.  How do you define defaults in the `type` declaration?

Reply:
I think you don't need `?` since it will always have a value but what I mean is making it required and without default value.

Reply:
it's an internal function where we always provide a value so it doesn't make much difference (the default was primarily to make changing the tests less annoying) but sure, can do.